examples/fluent-bit/multi-config-support/task-definition.json

```diff
@@ -0,0 +1,60 @@
+{
+	"family": "firelens-example-multi-config-support",
+	"taskRoleArn": "arn:aws:iam::xxxxxxxxxx:role/ecs_task_iam_role",
+	"executionRoleArn": "arn:aws:iam::xxxxxxxxxx:role/ecs_task_execution_role",
+	"containerDefinitions": [
+		{
+			"essential": true,
+			"image": "public.ecr.aws/aws-observability/aws-for-fluent-bit:init-latest",
+			"name": "log_router",
+			"firelensConfiguration": {
+				"type": "fluentbit"
+			},
+			"logConfiguration": {
+				"logDriver": "awslogs",
+				"options": {
+					"awslogs-group": "firelens-container",
+					"awslogs-region": "us-west-2",
+					"awslogs-create-group": "true",
+					"awslogs-stream-prefix": "firelens-init"
+				}
+			},
+            "environment": [
+                {
+                    "name": "aws_fluent_bit_init_s3_1",
+                    "value": "arn:aws:s3:::xxxxxxxxxx/dummy-input.conf"
+                },
+                {
+                  "name": "aws_fluent_bit_init_s3_2",
+                  "value": "arn:aws:s3:::xxxxxxxxxx/dummy-filter.conf"
+                },
+                {
+                  "name": "aws_fluent_bit_init_s3_3",
+                  "value": "arn:aws:s3:::xxxxxxxxxx/dummy-parser.conf"
```

Comment:
Since the tutorial text is already pretty long and gives you the config files directly, I think this part of the task def should also be  shown in the tutorial, which will help the user clearly see how init works with the env vars.